src/openstudio_app/OpenStudioApp.cpp

```diff
@@ -241,9 +245,9 @@ void OpenStudioApp::onMeasureManagerAndLibraryReady() {
       while (!measureManager().isStarted()) {
         LOG(Fatal, "Failed to start the Measure Manager on try " << currentTry << ", timeout reached.");
         QMessageBox msgBox;
-        msgBox.setWindowTitle("Timeout");
+        msgBox.setWindowTitle(tr("Timeout"));
         msgBox.setIcon(QMessageBox::Critical);
-        msgBox.setText("Failed to start the Measure Manager. Would you like to retry?");
+        msgBox.setText(tr("Failed to start the Measure Manager. Would you like to retry?"));
```

Comment:
Qt Linguist stores not only the original stirng, but also the filepath, class, and line number. So it should be able to update it just fine. Defining a variable wouldn't be terribly useful in this context.

lupdate/lrelease is ran via cmake on each build, so it stay current. From personal experience with this, you should commit (or at least stage) the OpenStudioApp_xx.ts file often, as when modifying a large number of lines (or worse, messing with the CMake settings regarding qt5_create_translations) it could sometimes have some hiccups.